index.html

```diff
@@ -9,21 +9,37 @@
   </head>
   <body>
     <!-- the header-section-->
-
     <header class="header">
       <div class="brand">
         <a href="https://github.com/EzekielUtshudi"> Welcome </a>
       </div>
       <nav class="navbar">
-        <a class="nav-link" href="/portfolio"> Portfolio </a>
-        <a class="nav-link" href="/about"> About </a>
-        <a class="nav-link" href="/contact"> Contact </a>
+        <a class="nav-link" href="#works"> Portfolio </a>
+        <a class="nav-link" href="#about"> About </a>
+        <a class="nav-link" href="#contact"> Contact </a>
       </nav>
-      <nav class="heading-manu">
-        <button type="button" class="mb-menu">
+
+      <nav class="hamburger">
+        <button type="button" class="mb-menu" aria-label="menu">
```

Comment:
- [ ] There's an issue with your closing button within the menu. It happens that the button is not being shown on the screen. I can easily click the open button (the hamburger button) but then have to guess where the close button is because I cannot see it on the screen. Please take a look at this issue as well.

Reply:
Thank you for your feedback, please I spend 3 days trying to solve that, is there anywhere I can get assisted to solve that ? because by me I don't see those errors